apps/wellknown/urls.py

```diff
@@ -10,6 +11,7 @@
 
 urlpatterns = [
     path("openid-configuration", openid_configuration, name="openid-configuration"),
+    path("smart-configuration", smart_configuration, name="smart-configuration"),
```

Comment:
the .well-known/smart-configuration should be relative to the base path of the server, eg v2/fhir/.well-known/smart-configuration for v2, v1/fhir/.well-known/smart-configuration for v1, per environment.

Reply:
I see what you mean. I'll have to do a bit more digging around to get it to have that pattern. Would we want to change openid-configuration to follow that pattern as well? It seems like it's the only one to have this current pattern. Swagger link for convenience: https://sandbox.bluebutton.cms.gov/docs/openapi#/v2/openIdConfig

Reply:
A good team discussion topic. imo the value of a well-known url is that you can automatically discover it. As it stands today, I'd see the same problem for the OIDC config, since openid-configuration-v2 isn't really well-known.

Reply:
Okay, I'll socialize this with the other engineers and see about making this happen